src/intersection-observer-polyfill.js

```diff
@@ -411,7 +413,9 @@ function calculateChangeEntry(
   }
 
   return /** @type {!IntersectionObserverEntry} */ ({
-    time: performance.now(),
+    time: (typeof performance !== 'undefined' &&
+        typeof performance.now !== 'undefined') ? performance.now()
```

Comment:
As @aghassemi  pointed out https://github.com/ampproject/amphtml/issues/6312#issuecomment-262398954

for performance.now check. yes it's probably unnecessary. However I see we are doing this check in other places. I thought it would be safer to include the extra check. I am ok with removing it.